doc/developers/contributing.rst

```diff
@@ -943,6 +943,15 @@ In iterative algorithms, the number of iterations should be specified by
 an integer called ``n_iter``.
 
 
+Pairwise Attributes
+^^^^^^^^^^^^^^^^^^^
+
+For estimators, such as those in :mod:`sklearn.svm` and 
+:class:`kernel_ridge.KernelRidge`, that use ``X`` with shape ``(n_samples, n_samples)``,
+there is a ``_pairwise`` attribute that allows cross-validation of the dataset.
```

Comment:
Maybe it would actually better to say "Estimators that accept X of shape .." need to specify ``"_pairwise=True"``